### DISCUSSION TITLE
Signing UI: Improving the PSBT case

### DISCUSSION THREAD

**Discussion by dennisreimann** [btcpayserver/btcpayserver#2652] [General]
As brought up in [this comment](https://github.com/btcpayserver/btcpayserver/pull/2559#issuecomment-860269369):

I think we can improve the PSBT case, because the flow is kind of weird when you are coming from the send view.
After `Send > PSBT` you get this:

![psbt](https://user-images.githubusercontent.com/886/124171916-7314c000-daa9-11eb-811a-e81fd6d894de.png)

Having "Sign transaction" as the primary action does not make sense for this case, as it would take you back to the signing options view you just saw. What you'd actually want to use is one of the "Other actions", most likely Download or Show QR (depending on your wallet). Once you've processed it with the signing wallet, you'll want to import it back into BTCPay and "Review" it.

I think we should separate the two cases depending on where one originates: When you start by importing a PSBT, having the "Sign transaction" option makes sense. For the Send flow I'd recommend we restructure this and pull the export actions out of the dropdown, make them primary and offer a clear way to afterwards import the resulting PSBT.

/cc @dstrukt

**Comment by dstrukt** [comment]
Agreed.

ACK. 

Very different use cases, and should be clear from the beginning of the flow.

**Comment by Kukks** [comment]
I would suggest a different approach: Showing all actions without a dropdown to have a kind of action bar. I would also rename the title as it's confusing (maybe just drop `Decoded`)

**Reply by bolatovumar** [reply]
I wonder if this would confuse users as now they have a bunch of different buttons showing up all at once and it's not clear what the next step is.
